seed/php-sdk/imdb/src/Core/Utils.php

```diff
@@ -39,4 +39,23 @@ public static function castKey(mixed $key, string $keyType): mixed
             default => $key,
         };
     }
+
+    /**
+     * Returns a human-readable representation of the input's type.
+     *
+     * @param mixed $input The input value to determine the type of.
+     * @return string A readable description of the input type.
+     */
+    public static function getReadableType(mixed $input): string
+    {
+        if (is_object($input)) {
+            return 'object(' . get_class($input) . ')';
+        } elseif (is_array($input)) {
+            return 'array(' . count($input) . ' items)';
```

Comment:
Could we resolve the the of the array from the first element (if any), and include that in the string representation, too? That way it could read like: `array<string>`.

Again, not actionable here since it's probably non-trivial to handle arbitrarily nested `array` types. Just things for us to consider moving forward.

Reply:
yeah agreed this would be great improvement 👍